ide/git/src/org/netbeans/modules/git/ui/history/RevisionNode.java

```diff
@@ -157,17 +193,22 @@ public PropertyEditor getPropertyEditor() {
         }
     }
     
-    private static String highlight (String author, String needle, String bgColor, String fgColor) {
+    private static String escape(String text) {
+        return text.replace("<", "&lt;");
```

Comment:
NetBeans has a more complete implementation that is already being used in `DiffTreeTable`

```suggestion
        try {
            return XMLUtil.toElementContent(text);
        } catch (CharConversionException e1) {
            Logger.getLogger(RevisionNode.class.getName()).log(Level.INFO, "Can not HTML escape: ", text);  //NOI18N
        }
```